tic_tac_toe_flutter/lib/core/design_system/tokens/spacing/app_spacing.dart

```diff
@@ -0,0 +1,8 @@
+final class AppSpacing {
+  const AppSpacing();
+  static const double s = 8;
+  static const double m = 16;
+  static const double l = 24;
+  static const double xl = 32;
+  static const double xxl = 40;
+}
```

Comment:
If `AppSpacing` never will be instantiated (i.e. you will never create an object of this type), it is better tu use the following way:

\`\`\`dart  
extension AppSpacing on Never {  
  static const double s = 8;  
  static const double m = 16;  
  static const double l = 24;  
  static const double xl = 32;  
  static const double xxl = 40;  
}  
\`\`\`